src/MetalPerformanceShaders/MPSKernel.cs

```diff
@@ -20,6 +20,18 @@ public static bool Supports (IMTLDevice device)
 			return MPSSupportsMTLDevice (device == null ? IntPtr.Zero : device.Handle);
 		}
 
+		[DllImport (Constants.MetalPerformanceShadersLibrary)]
+		[Introduced (PlatformName.MacCatalyst, 13, 0)]
+		[TV (12,2), Mac (10,14,4), iOS (12,2)]
+		static extern /* id<MTLDevice> _Nullable */ IntPtr MPSGetPreferredDevice (MPSDeviceOptions options);
```

Comment:
Is it standard practice to leave the `/*...*/` in code?

Reply:
Yes. we do that as a reference for the next person.

Reply:
The function just above it did, so I was copying that. I can go either way.

I hope we can get null annotations some day :-)